installbuilder/datafiles/Base_DSC.data

```diff
@@ -223,12 +223,13 @@ done
 chown -R ${{RUN_AS_USER}} /etc/opt/omi/conf/${{SHORT_NAME}}
 chown -R ${{RUN_AS_USER}} $OMI_REGISTER_DIR/root-oms
 /usr/sbin/usermod -g omiusers omsagent
+chown -R omsagent /opt/microsoft/omsagent/plugin
 
 # Set up the nxOMSAgent module
 /opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nx_1.0.zip 0
 /opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nxOMSAgent_1.0.zip 0
 /opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nxOMSCustomLog_1.0.zip 0
-/opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nxOMSPlugin_1.0.zip 0
+/opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nxOMSPlugin.zip 0
```

Comment:
Hi John. Thanks for pointing this out. I looked at InstallModule and it will not work. In SCRUM today we discussed that the signing job will be responsible stripping the version number and creating a properly signed name package to upload to Automation DSC.